hmpps-prisoner-search-indexer/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/model/translator.kt

```diff
@@ -67,8 +67,11 @@ fun Prisoner.translate(existingPrisoner: Prisoner? = null, ob: OffenderBooking,
       when (pm.type) {
         "Tattoo" -> this.tattoos = this.tattoos?.plus(bodyPart) ?: listOf(bodyPart)
         "Scar" -> this.scars = this.scars?.plus(bodyPart) ?: listOf(bodyPart)
-        "Mark" -> this.marks = this.marks?.plus(bodyPart) ?: listOf(bodyPart)
-        "Other" -> this.otherMarks = this.otherMarks?.plus(bodyPart) ?: listOf(bodyPart)
+        "Mark", "Other" -> {
+          this.marks = this.marks?.plus(bodyPart) ?: listOf(bodyPart)
+          this.tattoos = this.tattoos.addIfContains(bodyPart, "tattoo")
```

Comment:
why no `?` on `this.tattoos`?

Reply:
ah just spotted that `addIfContains` is a function defined below

Reply:
Because the extension function is on `List<BodyPartDetail>?`